test_runner/fixtures/neon_fixtures.py

```diff
@@ -756,6 +810,15 @@ def enable_real_s3_remote_storage(self, test_name: str, force_enable: bool = Tru
             prefix_in_bucket=self.remote_storage_prefix,
         )
 
+        if enable_remote_extensions:
+            self.ext_remote_storage = S3Storage(
```

Comment:
maybe we can parameterize this instead of using the same bucket in all regions?
dev buckets:
- eu-central-1
- eu-west-1
- us-east-2